torch/_dynamo/variables/nn_module.py

```diff
@@ -288,7 +288,10 @@ def call_function(
             # If we are tracing the higher order op, we want Dynamo to step
             # inside the module call so that Dynamo can see the underlying
             # parameters and buffers and raise them as inputs to the graph.
-            if tx.output.is_root_tracer() and mod.__module__.startswith("torch.nn."):
+            if tx.output.is_root_tracer() and (
+                mod.__module__.startswith("torch.nn.")
+                or mod.__module__.startswith("torch.ao.")
```

Comment:
```suggestion
                mod.__module__.startswith(("torch.nn.", "torch.ao."))
```

Reply:
Thanks for your suggestion! I incorporated this in reland of #116312.